google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java

```diff
@@ -378,6 +380,40 @@ public long writeFileToTable(String datasetName, String tableName, Path csvPath)
     // [END writeFileToTable]
   }
 
+  /**
+   * Example of writing a newline-delimited-json file with textual fields from GCS to a table.
+   */
+  // [TARGET writer(WriteChannelConfiguration)]
+  // [VARIABLE "my_dataset_name"]
+  // [VARIABLE "my_table_name"]
+  // [VARIABLE "my_source_uri")]
+  // [VARIABLE "field_names")]
```

Comment:
Likewise for `field_names`. The schema in the sample should be concrete. That is: don't pass it in, show it being constructed.

Aside: if we were to keep this argument (which we hopefully are not), the variable should be a list of strings, not just a string.

Reply:
Same as above.